src/game/attack/AttackActionHandler.ts

```diff
@@ -64,24 +64,26 @@ class AttackActionHandler {
 	 * Schedule an attack on an unclaimed territory.
 	 * @param player The player that is attacking.
 	 * @param troops The amount of troops that are attacking.
+	 * @param borderTiles The tiles from which the attack is executed. If it is null, it will be the player's border tiles by default.
 	 */
-	attackUnclaimed(player: Player, troops: number): void {
+	attackUnclaimed(player: Player, troops: number, borderTiles: Set<number> | null): void {
```

Comment:
`borderTiles` could have `null` as a default value, saving the need to retroactively pass `null` to `attackUnclaimed`

Reply:
i thought the same but player also passed as null. idk why platz made it like this but i made the same

Reply:
Adding a default value (of null) would be fitting here, since not specifying a border overwrite is the default case